addons/link_tracker/models/link_tracker.py

```diff
@@ -200,6 +200,14 @@ def action_visit_page(self):
             'target': 'new',
         }
 
+    def action_visit_page_statistics(self):
```

Comment:
Dependent of website_links where the route '+' (so much naming) is defined ... so to move in that module.

Reply:
@qmo-odoo Can you fix it ?